contributing.html

```diff
@@ -25,9 +25,8 @@
 
 To submit a change for inclusion please do the following:
 
-<ul>
-	<li>Create a patch that applies cleanly against <a href="http://svn.apache.org/repos/asf/kafka/site/">SVN trunk</a>.</li>
-	<li>Open a <a href="https://issues.apache.org/jira/browse/KAFKA">JIRA</a> ticket describing the patch and attach your patch to the JIRA. Include in the JIRA information about the issue and the approach you took in fixing it (if this isn't obvious). Also, mark the jira as "Patch Available" by clicking on the "Submit Patch" button (this is done automatically if you use the code review tool below). Creating the JIRA will automatically send an email to the developer email list alerting us of the new issue.</li>
+<ul>    
+         <li>Follow the detailed instructions in <a href="https://cwiki.apache.org/confluence/display/KAFKA/Contributing+Website+Documentation+Changes">Contributing Website Changes</a>.</li>
```

Comment:
Seems like this line is over-indented. Also, I'd probably drop `detailed` and just say "Follow the instructions..."